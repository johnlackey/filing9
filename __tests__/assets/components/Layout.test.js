import renderer from 'react-test-renderer'
import Layout from '../../../assets/components/Layout'

describe('<Layout>', function () {
  it('renders children in div', function () {
    const component = renderer.create(<Layout><span /></Layout>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
