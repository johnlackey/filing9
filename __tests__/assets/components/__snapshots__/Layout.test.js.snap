// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout> renders children in div 1`] = `
<div
  className="container"
>
  <span />
</div>
`;
